sessionctx/variable/session.go

```diff
@@ -1247,6 +1254,8 @@ func (s *SessionVars) SetSystemVar(name string, val string) error {
 		atomic.StoreUint64(&config.GetGlobalConfig().Log.QueryLogMaxLen, uint64(tidbOptInt64(val, logutil.DefaultQueryLogMaxLen)))
 	case TiDBCheckMb4ValueInUTF8:
 		config.GetGlobalConfig().CheckMb4ValueInUTF8 = TiDBOptOn(val)
+	case TiDBFoundInPlanCache:
```

Comment:
IMHO, this variable should not be set by user?

Reply:
Right. `last_statement_found_in_plan_cache` is a variable that only needs to be read. The set operator used for most system variables seems totally useless here and may cause some puzzles for users.